azkaban-common/src/main/java/azkaban/event/Event.java

```diff
@@ -41,6 +47,13 @@ private Event(Object runner, Type type, EventData eventData) {
     this.time = System.currentTimeMillis();
   }
 
+  private Event(Object runner, Type type) {
```

Comment:
code duplication.
reuse previous constructor rather than implementing the same logic again?